public/scripts/worktrollyUI.js

```diff
@@ -136,4 +137,21 @@ $("#submitNewSprint").click(function() {
         newPage = "dashboard";
         uiLoader();
     });
+});
+
+$("#logWorkSubmit").click(function() {
+    var sprintNumber = $("#sprintNumber").val();
+    var logTaskId = $("#logTaskId").val();
+    var logWorkDone = $("#logWorkDone").val();
+    var logWorkComment = $("#logWorkComment").val();
+    var logWorkStatus = $("#logWorkStatus").val();
+    logWorkDone = parseInt(logWorkDone) + parseInt(logWorkRT);
+
+    var logWorkFunction = firebase.functions().httpsCallable('logWork');
+    logWorkFunction({ SprintNumber: sprintNumber, LogTaskId: logTaskId, LogWorkDone: logWorkDone, LogWorkStatus: logWorkStatus, LogWorkComment: logWorkComment }).then(result => {
```

Comment:
Please correct sprintNumber variable name in create new task api, and also in the class structure.

If you are not understanding this please contact me.

Reply:
The time logWorkHours are not received by the user, please try to understand logWorkDone and logWorkHours are 2 different variables, we will take these inputs from the user. Improve the logic in the backend as well as in the frontend.